crates/rpc/src/handlers/config.rs

```diff
@@ -61,7 +44,34 @@ impl From<Arc<NetworkSpec>> for SpecConfig {
 }
 
 /// Called by `config/spec` to get specification configuration.
-pub async fn get_spec(network_spec: Arc<NetworkSpec>) -> Result<impl Reply, Rejection> {
+#[get("config/spec")]
+pub async fn get_config_spec(
+    network_spec: web::Data<Arc<NetworkSpec>>,
+) -> actix_web::Result<impl Responder> {
+    let network_spec = network_spec.as_ref();
+    let spec_config = SpecConfig::from(network_spec);
+
+    Ok(HttpResponse::Ok().json(spec_config))
+}
+#[get("config/spec1")]
+pub async fn get_config_spec1(
+    network_spec: web::Data<Arc<NetworkSpec>>,
+) -> actix_web::Result<impl Responder> {
+    let network_spec = network_spec.as_ref();
     let spec_config = SpecConfig::from(network_spec);
-    Ok(with_status(Data::json(spec_config), StatusCode::OK))
+
+    Ok(HttpResponse::Ok().json(spec_config))
+}
+
+/// Called by `/deposit_contract` to get the Genesis Config of Beacon Chain.
+#[get("config/deposit_contract")]
+pub async fn get_config_deposit_contract(
+    network_spec: web::Data<Arc<NetworkSpec>>,
+) -> actix_web::Result<impl Responder> {
+    let network_spec = network_spec.as_ref();
+    let deposit_contract = DepositContract::new(
+        network_spec.network.chain_id(),
+        network_spec.deposit_contract_address,
+    );
+    Ok(HttpResponse::Ok().json(deposit_contract))
```

Comment:
```suggestion
#[get("config/deposit_contract")]
pub async fn get_config_deposit_contract(
    network_spec: web::Data<Arc<NetworkSpec>>,
) -> actix_web::Result<impl Responder> {
    Ok(HttpResponse::Ok().json(Data::new(DepositContract::new(
        network_spec.network.chain_id(),
        network_spec.deposit_contract_address,
    ))))
```

It should be like this